JetSet/JetSet/WarthogDemoTodos.swift

```diff
@@ -7,14 +7,6 @@
 
 import SwiftUI
 
```

Comment:
ChecklistData was removed from the WarthogDemoTodos.Swift file, and instead stored in a new model file renamed "Checklist.swift".